bin/index.js

```diff
@@ -248,6 +249,119 @@ module.exports.createWorkflowRequest = function (token_filename, project_id) {
     });
 }
 
+module.exports.createRunRequest = function (token_filename, component_id, project_id) {
+    if (has_id_token(token_filename)) {
+        public_token_header = read_id_token(token_filename);
+    }
+    var payload = {
+        "component_details":{
+            "component_name": "calibration",
+            "component_id": component_id
+        },
+        "project_id": project_id,
+        "name": "Polly™ QuantFit Untitled"
```

Comment:
We should also be able to fetch the app name using a component id or similar info. Again, can be left for later.